### ISSUE TITLE
[1.12.2] Credit score resets on death

### ISSUE THREAD

**Issue by Morpheus1101** [kamildanak/EnderPay#11]
After having banked nearly 800 in credit an dying quite a few times, i noticed my credit balance had suddenly reset back to the starting amount.

**Comment by carstorm** [comment]
I believe this has to do with the way the config is set up rather than an issue in the code. Realizing this is what made me add the request outlined in Issue #12

**Comment by kamildanak** [comment]
Good point! The maximum resetLoginDelta was set to 100 days before 6383eb5fff3fce23f74c02d602abdbfd7ef8a38c (which I just pushed).
With day length set to 1h (default was 1 real life day) it would mean about 4 days before the credit balance would reset back to starting amount.

**Comment by Morpheus1101** [comment]
This actually makes sense now you mention that, as i had noticed the reset occurring once the balance reached a certain amount
